Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptWrapperAddController.java

```diff
@@ -82,10 +83,21 @@ private void initBinder(WebDataBinder binder) {
      *         page
      */
     @RequestMapping(value = "auth/conceptlist/addconceptwrapper")
-    public String prepareConceptWrapperAdd(@ModelAttribute ConceptWrapperAddBean conceptWrapperAddBean,
-            HttpServletRequest req, ModelMap model) {
+    public String prepareConceptWrapperAdd(@RequestParam(value = "wrapperId", required = false) String wrapperId,
+            HttpServletRequest req, ModelMap model){
         model.addAttribute("types", conceptWrapperService.fetchAllConceptTypes());
         model.addAttribute("lists", conceptWrapperService.fetchAllConceptLists());
+        ConceptEntry entry = null;
+        try {
+            entry = conceptManager.getWordnetConceptEntry(wrapperId);
+        } catch (LuceneException e) {
+            e.printStackTrace();
+        }
```

Comment:
this is not appropriate exception handling. First of all, you're not logging the exception; second, all references below to entry will throw a nullpointer now; third, what will the user see?

take a look at how this is done in other parts of the application.

Reply:
I have handled the exception by logging the appropriate message for the exception. The references below entry are not throwing any null pointer.